discord/bot.py

```diff
@@ -134,23 +136,41 @@ def on_info_command(self, event):
 
     @Plugin.command('stats', aliases=['statistics'], parser=True)
     @Plugin.parser.add_argument('username')
+    @Plugin.parser.add_argument('date', nargs='?')
```

Comment:
Issue here, probably caused by the nargs (also check other commands using the same pattern), date is recognized as the database